### PR TITLE
Convert examples to publishable templates

### PR DESCRIPTION
**Pull request by MarshallOfSound**
This is basically moving a few files around and making the exported config correctly for each template 👍

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,16 +1,16 @@
-## electron-forge examples
+## Electron Forge Templates
 
-This repo helps you to get started with many of the popular web frameworks in Electron, by providing a ready-to-go template based on [electron-forge](https://github.com/electron-userland/electron-forge/).
+This repo helps you to get started with many of the popular web frameworks in Electron, by providing a ready-to-go templates for [electron-forge](https://github.com/electron-userland/electron-forge/).
 
 ## How do I try these out?
 
-All of the repos have the same build scripts. `cd` into the directory you're interested in, and type this to get started:
+All of these templates are **built-in** to `electron-forge`.  All you have to do is install forge, initialize your project and get coding!
```
Comment (malept) [inline README.md:7]: How is the built-in part going to work? Does that just mean that they're available on NPM?

Comment (MarshallOfSound) [reply README.md:7]: Like this

https://github.com/electron-userland/electron-forge/pull/77 👍

**Comment by malept** [comment]
:+1: looking good but I want to see what the automated tests look like before hitting that big green approved button.

**Comment by MarshallOfSound** [comment]
Smoke tests are up 👍

**Comment by MarshallOfSound** [comment]
And tests are green 👍 

These tests purely ensure that a clean init doesn't blow up.  Whether the files are added / moved correctly is tested by `electron-forge`

**Review by malept — APPROVED** [review]

**Pull request merged**